clientv3/balancer.go

```diff
@@ -47,6 +47,15 @@ type simpleBalancer struct {
 	// upc closes when upEps transitions from empty to non-zero or the balancer closes.
 	upc chan struct{}
 
+	// downc closes when grpc calls down() on pinAddr
+	downc chan struct{}
+
+	// stopc closes when updateNotifyLoop() is done.
```

Comment:
this comment is incorrect; `stopc` is closed to signal updateNotifyLoop should stop